server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlVisitor.java

```diff
@@ -106,12 +106,21 @@ public List<Node> visitLob_xml_host_variables(Db2SqlParser.Lob_xml_host_variable
         if (ctx.xml_lobNO_size() != null) {
             addXmlLobNodes(variableDefinitionNode, generatedVariableLevel);
         } else if (ctx.lobWithSize() != null) {
-            addLobWithSizeNodes(variableDefinitionNode, generatedVariableLevel, lobSize(ctx.lobWithSize()));
+            addLobWithSizeNodes(variableDefinitionNode, generatedVariableLevel, lobSize(ctx.lobWithSize().lobSize()));
```

Comment:
Could the `lobSize()` return null if the input contains only e.g. `BLOB ` without the parenthesis following?

Reply:
I would expect that xml_lobNO_size will be not null, do you think it can be not always this way?

Reply:
After closer inspection, it won't fail because the non-execsql rules need to be "pre-matched" by antlr; so the parser will only produce valid contexts.